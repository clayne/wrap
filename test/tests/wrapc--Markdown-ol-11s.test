wrapc | /dev/null | -u | md-ol-11s.c | 0
