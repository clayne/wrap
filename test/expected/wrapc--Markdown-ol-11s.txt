  // 3. First sentence of third item.
  // 4. First sentence of fourth item.
  // 5. First sentence of fifth item.
  // 6. First sentence of sixth item.
  // 7. First sentence of seventh item.
  // 8. First sentence of eighth item.
  // 9. First sentence of nineth item.
  // 10. First sentence of tenth item.
  // 11. First sentence of eleventh item.
